From 9380d7f20486a286e191cb27f15ed19f99a1c4dc Mon Sep 17 00:00:00 2001
From: Thierry Strudel <tstrudel@google.com>
Date: Wed, 13 Jul 2016 09:58:11 -0700
Subject: [PATCH 083/420] irqchip: msm: log wakeup interrupts

set msm_show_resume_irq_mask to log wakeup reasons for batterystats

Bug: 28383208
Change-Id: I00b856c691d9da7ed6452704db9b0f3f56f49581
Signed-off-by: Thierry Strudel <tstrudel@google.com>
---
 drivers/irqchip/msm_show_resume_irq.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/irqchip/msm_show_resume_irq.c b/drivers/irqchip/msm_show_resume_irq.c
index 521149671fdf..cfadb7c2643d 100644
--- a/drivers/irqchip/msm_show_resume_irq.c
+++ b/drivers/irqchip/msm_show_resume_irq.c
@@ -15,7 +15,7 @@
 #include <linux/kernel.h>
 #include <linux/init.h>
 
-int msm_show_resume_irq_mask;
+int msm_show_resume_irq_mask = 1;
 
 module_param_named(
 	debug_mask, msm_show_resume_irq_mask, int, S_IRUGO | S_IWUSR | S_IWGRP
-- 
2.15.0

